models/booking.py

```diff
@@ -161,8 +161,10 @@ def mediationId(self):
     @property
     def qrCode(self):
         from domain.bookings import generate_qr_code
-        if not (self.isUsed or self.isCancelled):
-            return generate_qr_code(self)
+        offer = self.stock.resolvedOffer
+        if offer.isEvent:
+            return generate_qr_code(self) if not (self.isEventExpired or self.isCancelled) else None
```

Comment:
```suggestion
            return generate_qr_code(self) if self.isEventExpired is False and self.isCancelled is False else None
```
Qui me semble pour facile à lire, qu'en penses tu ?